src/splitInteger.test.js

```diff
@@ -8,12 +8,29 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(splitInteger(3, 3)).toBeInstanceOf(Array);
+  });
 
+  it(`should return an array with 'num' if parts = 1`, () => {
+    expect(splitInteger(3, 1)).toStrictEqual([3]);
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
+    expect(splitInteger(3, 1)).toStrictEqual([3]);
+  });
+
+  it(`there are parts elements (for example, if parts = 5, 
+    the num would be splitted into 5 integers)`, () => {
+    expect(splitInteger(0, 5)).toStrictEqual([0, 0, 0, 0, 0]);
+  });
+
+  it(`the integers in the result array should be 
+  in the ascending order`, () => {
+    expect(splitInteger(27, 13))
+      .toStrictEqual([2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 3]);
+  });
```

Comment:
1. ‘The input to your function will always be valid’, so you need to test only positive integers
2. The test should start with 'should return... if...'